ci-operator/config/openshift/loki/openshift-loki-upstream-v2.7.0-pre-ec0bf70.yaml

```diff
@@ -0,0 +1,57 @@
+base_images:
+  base:
+    name: "4.11"
+    namespace: ocp
+    tag: base
+  ocp_builder_rhel-8-golang-1.18-openshift-4.11:
+    name: builder
+    namespace: ocp
+    tag: rhel-8-golang-1.18-openshift-4.11
+build_root:
+  image_stream_tag:
+    name: release
+    namespace: openshift
+    tag: golang-1.18
+images:
+- dockerfile_path: Dockerfile.ocp
+  from: base
+  inputs:
+    ocp_builder_rhel-8-golang-1.18-openshift-4.11:
+      as:
+      - registry.ci.openshift.org/ocp/builder:rhel-8-golang-1.18-openshift-4.11
+  to: loki
+- dockerfile_path: Dockerfile.promtail.ocp
+  from: base
+  inputs:
+    ocp_builder_rhel-8-golang-1.18-openshift-4.11:
+      as:
+      - registry.ci.openshift.org/ocp/builder:rhel-8-golang-1.18-openshift-4.11
+  to: promtail
+promotion:
+  name: v2.7.0-pre-ec0bf70
```

Comment:
Does it make sense to leave out the commit hash and just call it `v2.7.0-pre`? (same for the branch, btw)